### DISCUSSION TITLE
Explorer les données de construction de nouveaux logements de SITADEL

### DISCUSSION THREAD

**Discussion by leabouche** [MTES-MCT/sparte#687] [General]
**Objectif:**
On cherche à croiser les données communales de logements vacants avec les données de construction de logements neufs. 

**Méthode**
Pour cela, on aimerait explorer les données de la base SITADEL, qui recense les données de permis de construire et d'autorisations d'urbanisme. (Liste des autorisations d'urbanisme créant des logements : https://www.data.gouv.fr/fr/datasets/base-des-permis-de-construire-et-autres-autorisations-durbanisme-sitadel/#/resources/1fa467ef-5e3a-456f-b961-be9032cfa3df)

Deux calculs d'intérêt:
- Nombre de nouveaux logements créés par commune et par an 
- Surface totale des nouveaux logements créés par commune et par an

**Critères** 

- Localisation
            - ADR_CODPOST_TER (Code postal du terrain) 
                      Le code de la commune est renseigné dans le code géographie de l'année de la demande d'AU ; les numero de parcelle ne sont pas uniques 

- Date à prendre en compte : 
            - DATE_REELLE_DAACT (date réelle d'achèvement des travaux) _ou bien_ 
            - DATE_REELLE_DOC (date réelle d'ouverture de chantier) ?
            
- Distinction parc privé / parc public : 
            - CAT_DEM (catégorie du demandeur = maitre d'ouvrage selon SITADEL) : pour correspondre au parc de logements privé et pouvoir être comparée aux données de logements vacants : exclure les codes 20;21,40;41;42;43 _- à confirmer_
           
-   Nature des travaux: 
             - NATURE_PROJET_DECLAREE = 1 (nouvelle construction)

- Décompte des logements 
            - NB_LGT_TOT_CREES (nombre total de logements créés)
                     - Peut être distingué en logement individuels (maison) vs. logement collectif (immeubles)
               
- Décompte surfaces
            - SURF_HAB_CREEE (Surface de plancher de la destination 'Habitation' nouvelle construite)

**Comment by alexisig** [comment]
Top que tu aies trouvé la catégorie de demandeur !

Je me demande à quel point on devrait aussi regarder les permis de démolir (toujours Sitadel) pour identifier et exclure les cas de _démolition puis construction_. Dans un soucis de simplicité ça pourrait être uniquement de faire construction - démolition à l'échelle de la commune dans un premier temps.

Les données d'instruction sont localisées sur 1 à 3 parcelles. Qu'est-ce que tu penses d'une carto sur ces données ?
Les parcelles sont disponibles en open data : https://www.data.gouv.fr/fr/datasets/parcellaire-express-pci/

**Reply by leabouche** [reply]
- Catégorie de demandeur : ce sont mes hypothèses, à confirmer :) 

- Démolitions: oui bon point! Dans ce fichier on a une variable NB_LGT_DEMOLIS qu'on pourrait utiliser directement (bien que marquée 'insuffisamment renseigné"..)
En fait dans l'absolu il faudra qu'on fasse le rapport des surfaces: consommation pour l'habitat / nouvelles constructions

- Carto des logements construits à la parcelle : pour le moment je vois pas bien ce que ça nous permettrait de mettre en lumière ?  En revanche à terme si on a aussi des données en infra comm pour les logements vacants on pourrait faire des croisements par zonage hyper intéressants !

**Comment by leabouche** [comment]
https://docs.google.com/spreadsheets/d/1RcubbfvlsUtSohst8Ksfvcse7WQtUxpN/edit?gid=102906318#gid=102906318

J'ai mis en rouge les variables qui me paraissent pertinentes pour notre usage et en jaune celles qu'on pourrait potentiellement utiliser
